specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -2470,6 +3142,43 @@
       ],
       "type": "object"
     },
+    "ConversationTasksState": {
+      "properties": {
+        "tasks": {
+          "properties": {
+            "completed": {
+              "type": "integer"
```

Comment:
Documenting these - even copying from another task state model - would be helpful for customers. Not blocking for the PR, but should ideally be added at some point.

Reply:
adding for new APIs